### ISSUE TITLE
Update to Electron 32

### ISSUE THREAD

**Issue by MRayermannMSFT** [microsoft/AzureStorageExplorer#8109]

**Comment by richardMSFT** [comment]
Updating to Electron 32 is blocked by https://github.com/electron/electron/pull/43555. The issue appears to have been addressed and a fix has been merged for 32.x.y, but we're waiting for a backport to 32.x.y: https://github.com/electron/electron/pull/43785.

**Comment by craxal** [comment]
Fix is now available in Electron 32, but it's at the end of the milestone, so we'll postpone upgrades till 1.37.0.

**Comment by MRayermannMSFT** [comment]
Updated issue to be for Electron 33. Stable should be available on 2024-Oct-15.

https://www.electronjs.org/docs/latest/tutorial/electron-timelines
